src/components/Navbar/styles.module.scss

```diff
@@ -48,37 +46,33 @@
     }
 
     .desktop_navbar {
-      display: block;
       @include flex_container(row, space-between, center);
+      max-width: 1400px;
```

Comment:
En mi pantalla se ve muy pegado, pondría un 1200px. stas maquetando teniendo en cuenta la resolución que te pasé?

Reply:
Podrías agregar otra media query, para pantalla mas grandes donde el max-width si sea de 1400px.
Por ejemplo:
`@media only screen and (min-width: 1400px) {`